rwgame/GameBase.cpp

```diff
@@ -1,26 +1,28 @@
 #include "GameBase.hpp"
+
+#include <rw/defines.hpp>
+#include <rw/filesystem.hpp>
+
 #include "GitSHA1.h"
 
 #include "SDL.h"
 
 // Use first 8 chars of git hash as the build string
 const std::string kBuildStr(kGitSHA1Hash, 8);
 const std::string kWindowTitle = "RWGame";
+const std::string kDefaultConfigFileName = "openrw.ini";
 constexpr int kWindowWidth = 800;
 constexpr int kWindowHeight = 600;
 
-GameBase::GameBase(Logger &inlog, int argc, char *argv[]) : log(inlog) {
+GameBase::GameBase(Logger &inlog, int argc, char *argv[]) :
+        log(inlog),
+        config(),
```

Comment:
Why put these empty constructors in the initaliser list?

Reply:
I always try to be explicit about these things.